infra/azure/README.md

```diff
@@ -128,9 +131,8 @@ Directory user's object ID.
 ./bootstrap.sh bootstrap <REPO>/hail:<BRANCH> create_initial_user <USERNAME> <OBJECT_ID>
 ```
 
-Deploy the gateway service. First trim down `$HAIL/letsencrypt/subdomains.txt`
-to only the services that are deployed and then run
+Deploy the gateway:
 
 ```
-make -C $HAIL/gateway deploy
+make -C $HAIL/gateway envoy-xds-config deploy
```

Comment:
Ah I see. You didn't fold `envoy-xds-config` into `deploy`. Why is that?

Reply:
Addressed in above comment. Happy to take any suggestions. The other option here is to go back to what I had before where `deploy` means "do everything" and there's another special target that does not overwrite the config. But this felt to me like the "safer" option that is less of a footgun.

Reply:
See comment above.